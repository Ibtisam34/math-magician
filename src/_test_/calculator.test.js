import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../Components/Calculator';

it('renders correctly', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
