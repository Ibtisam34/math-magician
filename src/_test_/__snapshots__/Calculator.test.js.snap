// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="calculator"
>
  <div
    className="display"
  >
    0
  </div>
  <div
    className="button-panel"
  >
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      รท
    </button>
    <button
      className="orange"
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="orange"
      id="zero"
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="black"
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="grey"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;